src/test/java/ScattTest.java

```diff
@@ -9,13 +13,68 @@
  * @author Erik Cole
  */
 public class ScattTest {
+    /**
+     * Get a string representing a path to a project test resource.
+     * Project test resources should be put in src/test/resources.
+     * @param resName The name of the resource file
+     * @return String representation of path to resource
+     */
+    private String getTestResource(String resName) {
+        URL url = this.getClass().getResource("/" + resName);
+        String filePath = url.getFile();
+        return filePath;
+    }
+    /**
+     * Test sb2 constructor with valid file path.
+     */
+    @Test
+    public void testSb2Ctor1() {
```

Comment:
The rest of your tests have rather verbose names that are super readable. Could we rename this to be `testSb2Constructor1`? I misread this abbreviation almost every time that I see it.